go.mod

```diff
@@ -80,3 +80,5 @@ replace (
 	github.com/go-openapi/errors => github.com/go-openapi/errors v0.19.4
 	github.com/go-openapi/validate => github.com/go-openapi/validate v0.19.8
 )
+
```

Comment:
please add a comment about why this was pinned

Reply:
Added explanation in acaf4da2ab086438698580aba97ec63640d12624